map/Region.dhall

```diff
@@ -0,0 +1,257 @@
+                      , "ap-northeast-2c"
+                      ]
+                    , ap-northeast-3 =
+                      [ "ap-northeast-3a"
+                      , "ap-northeast-3b"
+                      , "ap-northeast-3c"
+                      ]
+                    , ap-south-1 =
+                      [ "ap-south-1a", "ap-south-1b", "ap-south-1c" ]
+                    , ap-southeast-1 =
+                      [ "ap-southeast-1a"
+                      , "ap-southeast-1b"
+                      , "ap-southeast-1c"
+                      ]
+                    , ap-southeast-2 =
+                      [ "ap-southeast-2a"
+                      , "ap-southeast-2b"
+                      , "ap-southeast-2c"
+                      ]
+                    , ca-central-1 =
+                      [ "ca-central-1a", "ca-central-1b", "ca-central-1c" ]
+                    , cn-north-1 = [ "cn-north-1a", "cn-north-1b" ]
+                    , cn-northwest-1 =
+                      [ "cn-northwest-1a"
+                      , "cn-northwest-1b"
+                      , "cn-northwest-1c"
+                      ]
+                    , eu-central-1 =
+                      [ "eu-central-1a", "eu-central-1b", "eu-central-1c" ]
+                    , eu-north-1 =
+                      [ "eu-north-1a", "eu-north-1b", "eu-north-1c" ]
+                    , eu-south-1 =
+                      [ "eu-south-1a", "eu-south-1b", "eu-south-1c" ]
+                    , eu-west-1 = [ "eu-west-1a", "eu-west-1b", "eu-west-1c" ]
+                    , eu-west-2 = [ "eu-west-2a", "eu-west-2b", "eu-west-2c" ]
+                    , eu-west-3 = [ "eu-west-3a", "eu-west-3b", "eu-west-3c" ]
+                    , me-south-1 =
+                      [ "me-south-1a", "me-south-1b", "me-south-1c" ]
+                    , sa-east-1 = [ "sa-east-1a", "sa-east-1b", "sa-east-1c" ]
+                    , us-gov-east-1 =
+                      [ "us-gov-east-1a", "us-gov-east-1b", "us-gov-east-1c" ]
+                    , us-gov-west-1 =
+                      [ "us-gov-west-1a", "us-gov-west-1b", "us-gov-west-1c" ]
+                    , us-east-1 =
+                      [ "us-east-1a"
+                      , "us-east-1b"
+                      , "us-east-1c"
+                      , "us-east-1d"
+                      , "us-east-1e"
+                      , "us-east-1f"
+                      ]
+                    , us-east-2 = [ "us-east-2a", "us-east-2b", "us-east-2c" ]
+                    , us-west-1 =
+                          [ "us-west-1a", "us-west-1b" ]
+                        # ( if    _params.newCustomer.us-west-1
+                            then  [] : List Text
+                            else  [ "us-west-1c" ]
+                          )
+                    , us-west-2 =
+                      [ "us-west-2a", "us-west-2b", "us-west-2c", "us-west-2d" ]
+                    }
+                    region
+
+          in  render
```

Comment:
`{ function = render, Params }` so that the `Params` is actually accessible down-stream